docs/start/getting-started/fragments/ios/add-api.md

```diff
@@ -9,30 +10,29 @@ Now that you have DataStore persisting data locally, in the next step you'll con
 
    This command will open up a web browser to the AWS Management Console and guide you through creating a new IAM user. For step-by-step directions to set this up, refer to the [CLI installation guide](~/cli/start/install.md).
 
-1. Next, push your new API to AWS. To do this, **update the amplifytools.xcconfig** by changing `push=false` to:
+1. Next, push your new API to AWS. To do this, **run the command**:
     ```bash
-    push=true
+    amplify push
```

Comment:
This step failed for me. Looks like the script is doing more than just run "amplify push". Is it also doing an "init" by any chance?

Reply:
I updated it to include `amplify init` as a required step before `amplify push`